packages/common/src/search/_internal/portalSearchUsers.ts

```diff
@@ -116,6 +193,35 @@ async function searchPortal(
   };
 }
 
+/**
+ * Community search, which then converts `IGroup`s to `IHubSearchResult`s
+ * handling enrichments & includes along the way
+ *
+ * @param searchOptions
+ * @returns a promise that resolves enriched community user search results
+ */
+async function searchCommunity(
```

Comment:
New community user search method, mapped to `communityUser` `targetEntity`, does not force the org enrichment